src/SourceBuild/content/eng/pipelines/templates/jobs/sdk-diff-tests.yml

```diff
@@ -130,7 +130,7 @@ jobs:
       /p:MsftSdkTarballPath=$(MsftSdkTarballPath)
       /p:SdkTarballPath=$(SdkTarballPath)
       /p:SourceBuiltArtifactsPath=$(SourceBuiltArtifactsPath)
-      /p:SmokeTestsWarnOnSdkContentDiffs=false
+      /p:SourceBuildTestsWarnOnSdkContentDiffs=false

```

Comment:
Regarding [Nikola's previous comment](https://github.com/dotnet/sdk/pull/44514/files#r1821624475), this flag is still being used here. If this flag is no longer is use, we should consider either filing an issue to remove this flag or cleaning up the rest of it in this PR.

Reply:
Nikola's comment was for the primary CI/PR pipeline.  The flag is not used/necessary in that context as the sdk content diff tests are not run so I removed the flag from the run tests command.

It is useful in this context although we are specifying the default value and therefore could remove the warn support all together for the license and content diff tests.  It is low priority but may be worth tracking - I will open an issue.

Reply:
https://github.com/dotnet/source-build/issues/4709